docs/decisions/0007-adopt-vue-recommended-linting.md

```diff
@@ -0,0 +1,53 @@
+# 0007. Adopt vue/recommended linting
+
+Date: 2024-02-26
+Status: Proposed
+
+## Context and Problem Statement
+
+Our frontend code currently enforces the [essential](https://eslint.vuejs.org/rules/#priority-a-essential-error-prevention) 
+set of linting rules from [`eslint-plugin-vue`](https://eslint.vuejs.org/). This minimal set of 
+rules leaves a lot of code style up to individual authors and avoids enforcing a number of 
+community best practices. For this codebase to better support new volunteers frequently 
```

Comment:
👍 Definitely agree that putting more guardrails in place like this could ease the experience for new volunteers getting started